momento-sdk/src/intTest/java/momento/sdk/SimpleCacheClientTest.java

```diff
@@ -55,21 +53,21 @@ final class SimpleCacheClientTest extends BaseTestClass {
   void setup() {
     target =
         SimpleCacheClient.builder(System.getenv("TEST_AUTH_TOKEN"), DEFAULT_TTL_SECONDS).build();
+    target.createCache(cacheName);
   }
 
   @AfterEach
   void teardown() {
+    target.deleteCache(cacheName);
     target.close();
   }
 
   @Test
   public void createCacheGetSetDeleteValuesAndDeleteCache() {
-    final String cacheName = randomString("name");
```

Comment:
I don't know why these were random instead of using a static name or the TEST_CACHE_NAME env var that is supposed to be provided. I don't think the tests run simultaneously. @cprice404 do you know?